spec/messages/channel_spec.rb

```diff
@@ -52,4 +52,78 @@
       end
     end
   end
+
+  describe '#online' do
+    let(:online_users_body) do
```

Comment:
also, this seems more like it'd be better named `online_users_response` given that it includes both the body and status (same below)